gnosis/cowswap
# Add affiliate top link and affiliate base page

#848

Add Profile top link and affiliate base page

![image](https://user-images.githubusercontent.com/79759622/125974296-33204495-d8e1-482c-a2e8-ddc05b52f61c.png)
![image](https://user-images.githubusercontent.com/79759622/125974368-78858ba6-ed2d-4c35-8245-4a41201bdc76.png)


# Testing steps

1. Open the app
 - [x] In the top menu, there should be a link to the Profile page
2. Click on Rewards
 - [x] It should lead you to the rewards page
 - [x] Rewards page should match Figma design
